_includes/footer.html

```diff
@@ -3,9 +3,9 @@
     <div class="column d-4-12 m-2-12">
       <h4>For Competitors</h4>
       <ul class="sitemap">
+        <li><a href="/teams/">Teams Dashboard</a></li>
```

Comment:
(drive-by): I think this needs `site.baseurl` to ensure it works correctly locally:
```suggestion
        <li><a href="{{ '/teams/' | prepend: site.baseurl }}">Teams Dashboard</a></li>
```